bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -304,19 +304,14 @@ impl Client {
         initial_device_name: Option<String>,
         device_id: Option<String>,
     ) -> Result<(), ClientError> {
-
```

Comment:
Are you sure it wasn't only complaining about this first one? The others were actually useful for readability.

Reply:
Youre right, it was only complaining about the first space. Hm, i redid with the readability and submitted it. 
Thanks for the help!

Reply:
Can you verify now? I have made those changes

Reply:
Yep yep, looks good to me 👍